inc/datahandlers/user.php

```diff
@@ -1783,4 +1783,61 @@ function clear_profile($delete_uids=false, $gid=0)
 			remove_avatars($uid);
 		}
 	}
+
+	public function verify_signature()
+	{
+		global $mybb, $parser;
+
+		$parser_options = array(
+			'allow_html' => $mybb->settings['sightml'],
+			'filter_badwords' => 1,
+			'allow_mycode' => $mybb->settings['sigmycode'],
+			'allow_smilies' => $mybb->settings['sigsmilies'],
+			'allow_imgcode' => $mybb->settings['sigimgcode'],
+			"filter_badwords" => 1
+		);
+
+		if($this->data['showimages'] != 1 && $this->data['uid'] != 0)
```

Comment:
I don't really know how you can call it "careless copy+paste", because as far as I was aware there was no issue with the current code to be moved over regarding this comment or for me to look into this further. If there was an issue brought to attention, considering this has been here for a while, I could've looked into it and changed it appropriately.